### PR TITLE
feat: new replay errors for bad callbacks

### PR DESCRIPTION
**Pull request by LarsLiden**

### CODE REVIEW

File: src/ReplayError.ts [hunk 1]
```diff
@@ -7,6 +7,10 @@ export enum ReplayErrorType {
   ActionUndefined = 'ActionUndefined',
   /* Entity does not exist in the model */
   EntityUndefined = 'EntityUndefined',
+  /* API bad Card */
+  APIBadCard = 'APIBadCard',
+  /* API returns logicValue but has not Render */
```
Comment (mattmazzola) [inline src/ReplayError.ts]: minor typo, maybe not worth to fix since you have to rerun builds:
"has no Render function" or "does not have render()"

Comment (mattmazzola) [reply src/ReplayError.ts]: oh, disregard, already merged

**Comment by LarsLiden** [comment]
Cool.  Will make change locally and check in next time there's something
major

**Pull request merged**